### ISSUE TITLE
Hunyuan Lora output garbage no matter what

### ISSUE THREAD

**Issue by jutochoppa** [kohya-ss/musubi-tuner#86]
This is my third attempt with the same results. the output of my lora is always garbage

![Image](https://github.com/user-attachments/assets/f82a9880-8140-4242-9a7f-1762b2e13324)

the main training parameters I use are 
`
    accelerate launch --num_processes=1 --num_cpu_threads_per_process 1 --mixed_precision bf16 hv_train_network.py \
        --dit ... \
        --dataset_config ... \
        --save_state \
        --vae_chunk_size 64 --mixed_precision bf16  --sdpa --fp8_base\
        --optimizer_type adamw8bit --learning_rate 4e-4 --gradient_checkpointing \
        --max_data_loader_n_workers 4 --persistent_data_loader_workers \
        --network_module networks.lora --network_dim 32 --network_args "loraplus_lr_ratio=4"  \
        --timestep_sampling shift --discrete_flow_shift 3.0 \
        --max_train_epochs 1000 --save_every_n_epochs 1 --seed 420 \
        --output_dir ... \
        --output_name ... \
        --logging_dir logs`

I've also tried different hunyuan node types and workflows on comfyui without success

I have converted the lora with no success.

**Comment by jutochoppa** [comment]
Temp closing
